test/test_api_responses.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+from unittest.mock import MagicMock, patch
+
+import pytest
+import titan_client
+from .test_params import test_params
+
+prefix = os.path.abspath(os.path.dirname(__file__))
+
+class TestAPIResponses:
```

Comment:
Let's keep the structure flat as classes do not give any benefit here. They just introduce unnecessary complexity.
So let `test_api_responses` function be a first level member of `test/test_api_responses.py`